base/intfuncs.jl

```diff
@@ -638,6 +638,31 @@ function ndigits0z(x::Integer, b::Integer)
     end
 end
 
+# the suffix "2" means base 2
+"""
+    ndigits0z2(x::Integer) -> Integer
+
+Efficiently compute `ndigits0z(x, 2)`. This is the number of bits used to represent `x` in
```

Comment:
The first sentence isn't accurate, as they agree only for nonnegative numbers. 
```
julia> Base.ndigits0z(-10, 2)
4

julia> ndigits0z2(-10)
64
```
There is an example showing this, but best if the words aren't wrong.

Reply:
Thanks! Good catch. This is actually worse than a documentation error because `ndigits0z2(big(-10)) == 4 != ndigits0z2(-10)`.

Reply:
I think that deleting the generic fallback means this function can't solve the original motivating problem:

https://github.com/JuliaLang/julia/pull/47519/files#diff-8a30f80e1bd3e2cedf6ca2fb06681ce8257bd55d7f92b75432e049f569321d1aR314-R316

I didn't notice earlier that the analogy with `Base.ndigits0z` had led to a different fallback. It doesn't matter for that use since there's an `abs`.

Reply:
It can still help https://github.com/JuliaLang/julia/pull/47519/files#r1020836505.

You are right, though, that adding a generic fallback here would be able to replace the function defined in #47519 entirely. I think I'll wait for a name to come bac from triage before proceeding with the generic fallback.

Reply:
I put back the generic fallback.